file-formats/sajc/type/zif_aff_sajc_v1.intf.abap

```diff
@@ -37,20 +50,105 @@ INTERFACE zif_aff_sajc_v1
       "! <p class="shorttext">Notification</p>
       "! Name of the class which contains the notification exit
       notification TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Delete</p>
+      "! Name of the class which contains the delete exit
+      delete       TYPE zif_aff_types_v1=>ty_object_name_30,
     END OF ty_exit_classes.
 
+  "! <p class="shorttext">Parameter Name</p>
+  "! Name of the parameter
+  TYPES ty_parameter_name TYPE c LENGTH 38.
+  "! <p class="shorttext">Group Name</p>
+  "! Name of the group
+  TYPES ty_group_name TYPE c LENGTH 10.
+  "! <p class="shorttext">Section Name</p>
+  "! Name of the section
+  TYPES ty_section_name TYPE c LENGTH 10.
+  "! <p class="shorttext">Title Text</p>
+  "! Text of the title
+  TYPES ty_title_text TYPE c LENGTH 120.
+
   TYPES:
-    "! <p class="shorttext">General Information</p>
-    "! General information
-    BEGIN OF ty_general_information,
-      "! <p class="shorttext">Class with Execute Method</p>
-      "! Name of the class which contains the execute-method to run within the job
+    "! <p class="shorttext">Parameter</p>
+    "! Parameter of the class which is executed within the job
+    BEGIN OF ty_parameter,
+      "! <p class="shorttext">Name</p>
+      "! Name of the parameter
       "! $required
-      class_name   TYPE zif_aff_types_v1=>ty_object_name_30,
-      "! <p class="shorttext">Executed Program</p>
-      "! Name of the program which is executed within the job
-      program_name TYPE ty_program_name,
-    END OF ty_general_information.
+      name                 TYPE ty_parameter_name,
+      "! <p class="shorttext">Group</p>
+      "! Name of the parameter group
+      group                TYPE ty_group_name,
+      "! <p class="shorttext">Indent</p>
+      "! Flag indicating whether the parameter is indented on the selection screen
+      indent               TYPE abap_bool,
+      "! <p class="shorttext">Mandatory</p>
+      "! Flag indicating whether the parameter is mandatory
+      mandatory            TYPE abap_bool,
+      "! <p class="shorttext">Hidden</p>
+      "! Flag indicating whether the parameter is hidden
+      hidden               TYPE abap_bool,
+      "! <p class="shorttext">Read Only</p>
+      "! Flag indicating whether the parameter is read only
+      read_only            TYPE abap_bool,
```

Comment:
Minor: Is the order of the fields important? If it is not important, I would consider to group all boolean values and put them to the end of the structure.

Reply:
I agree, grouping would be nice in this case. I have to check this with the team.